Add t.me/useTGxBot (#59)
grammyjs/awesome-grammY: README.md

- [binamralamsal/WordSeek](https://github.com/binamralamsal/WordSeek) - A fun and competitive Wordle-style game that you can play with friends. [ᴜsᴇ](https://t.me/WordSeekBot)
- [Burhanverse/rssify](https://github.com/Burhanverse/rssify) - A feature-rich RSS bot with support for forum topics.
- [Burhanverse/tunified](https://github.com/Burhanverse/Tunified) - A Telegram bot to fetch the currently playing song from Last.fm and shares it on Telegram.
- [Taofeekabdulazeez/word-ninjas](https://github.com/Taofeekabdulazeez/word-ninjas) - A game where multiple players compete to solve anagrams in real-time.

> - [SpEcHiDe/JsonBot](https://github.com/SpEcHiDe/JsonBot) - A JSON bot (updated with the latest Bot API features) with ability to send the JSON of most of the updates generated by Telegram Bot API.

Reviewer: ```suggestion
- [SpEcHiDe/JsonBot](https://github.com/SpEcHiDe/JsonBot) - A bot that sends the JSON representations of most of the updates of the Bot API.
```
